sound/soc/soc-core.c

```diff
@@ -491,6 +491,14 @@ static struct snd_soc_pcm_runtime *soc_new_pcm_runtime(
 	if (!rtd->codec_dais)
 		goto free_rtd;
 
+	/*
+	 * for rtd->cpu_dais
+	 */
```

Comment:
This comment is not very helpful at all

Reply:
This is to be consistent with the existing code. ie
```
	/*
	 * for rtd->codec_dais
	 */
```
above.